.dockerignore

```diff
@@ -4,3 +4,5 @@
 
 *
 !requirements/
+!poetry.lock
```

Comment:
Could you please remind me what was the purpose of ignoring certain files?

Reply:
This was the error `Service 'backend' failed to build: COPY failed: stat /var/lib/docker/tmp/docker-builder600520132/poetry.lock: no such file or directory`
To avoid it I added it.